editor_constants.py

```diff
@@ -39,7 +39,7 @@
 
 WORKING_DIR = Path.cwd()
 if b_e_c.IS_WIN:
-    SETTINGS_DIR = WORKING_DIR
+    SETTINGS_DIR = Path(WORKING_DIR, "mek_config")
 else:
     SETTINGS_DIR = Path(Path.home(), ".local", "share", "mek", "mozzarilla")

```

Comment:
The SETTINGS_DIR on non-windows systems should mirror the windows one, and be Path(Path.home(), ".local", "share", "mek_config")